### DISCUSSION TITLE
__hostname__ 这个全局tag 可以改成可选的吗

### DISCUSSION THREAD

**Discussion by snakorse** [alibaba/ilogtail#865] [Ideas]
https://github.com/alibaba/ilogtail/blob/61a0579c2979c28ba5eed51f1d64706f40f69679/pluginmanager/plugin_runner_helper.go#L80

如上代码，在某些场景下可能不需要添加这个__hostname__的tag，比如agent在server端作为collector gateway部署时，可能期望不追加这个tag，原因：
1. 不关心是哪个server 节点处理的数据
2. 添加该tag，会导致透传到后端存储的时间序列，维度膨胀N倍(N为server段collector agent节点数)

**Comment by snakorse** [comment]
可能的改成可选的方案：GlobalConfig 添加一个选项：AddHostInfoTags bool， 默认为true

**Comment by henryzhx8** [comment]
短期内的话，converter里的TagKeyRenameMap参数可以干这个事情，而且不限于hostname这个

**Reply by snakorse** [reply]
> 短期内的话，converter里的TagKeyRenameMap参数可以干这个事情，而且不限于hostname这个

我们是自定义的外部flusher插件，可能用不了这个机制
